addons/website_mass_mailing_sms/views/snippets_templates.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <template id="s_newsletter_block_sms_template" groups="base.group_user">
```

Comment:
Note that there is a ``snippets`` directory already existing, shouldn't it be there ? What is the existing snippet by the way ?